apps/api/src/domain/models/__tests__/buildResourceModelContextStateValidatorTestCases/buildBibliographicReferenceResourceModelContextStateValidatorTestCase.ts

```diff
@@ -0,0 +1,10 @@
+import { resourceTypes } from '../../../types/resourceTypes';
+import buildAllInvalidTestCasesForResource from '../utilities/buildAllInvalidTestCasesForResource';
+import buildAllValidTestCasesForResource from '../utilities/buildAllValidTestCasesForResource';
+
+const validCases = buildAllValidTestCasesForResource(resourceTypes.bibliographicReference);
+
+export default () => ({
+    validCases,
+    invalidCases: [...buildAllInvalidTestCasesForResource(resourceTypes.bibliographicReference)],
```

Comment:
We should rename the helper that is called here to `buildAllInconsistentContextTypeTestCasesForResource`. The current name is misleading; you still need additional invalid cases for the context types your model supports. Add these.